tests/textcode/test_analysis.py

```diff
@@ -38,7 +38,12 @@
 from textcode.analysis import unicode_text_lines
 from textcode.analysis import numbered_text_lines
 from commoncode.fileutils import resource_iter
+from commoncode import compat
```

Comment:
It would be great to sort imports (they were not sorted before, that's a mistake to fix)